src/api/migrations/0005_auto_20221006_2037.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 3.2.15 on 2022-10-06 20:37
+
+from django.db import migrations
+
+
+def fix_mono(apps, schema_editor):
+    FishFamily = apps.get_model("api", "FishFamily")
+    FishGenus = apps.get_model("api", "FishGenus")
+
+    correct_family = FishFamily.objects.get(name="Monacanthidae")
+    incorrect_family = FishFamily.objects.get(name="Monocanthidae")
```

Comment:
should handle the case if the migration get's ran more than once (i.e. incorrect_family doesn't exist)